locations/spiders/eurochange_gb.py

```diff
@@ -0,0 +1,29 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class WilkoGBSpider(SitemapSpider, StructuredDataSpider):
+    name = "eurochange_gb"
+    EUROCHANGE = {"brand": "Eurochange", "brand_wikidata": "Q86525249"}
+    NM_MONEY = {"brand": "NM Money", "brand_wikidata": "Q86529747"}
+    item_attributes = EUROCHANGE
+    allowed_domains = ["www.eurochange.co.uk"]
+    sitemap_urls = ["https://www.eurochange.co.uk/sitemap.xml"]
+    sitemap_rules = [
+        (
+            r"https:\/\/www\.eurochange\.co\.uk\/branches\/([\w-]+\/[\w-]+)$",
+            "parse_sd",
+        ),
+    ]
+    wanted_types = ["Store"]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        if "facebook" in item.keys() and item["facebook"] == "https://www.facebook.com/eurochange/":
+            item["facebook"] = None
+        if "image" in item.keys() and item["image"] == "https://www.eurochange.co.uk/assets/img/logo.svg":
+            item["image"] = None
+        if "NM Money" in item["name"]:
+            item.update(self.NM_MONEY)
+            item["website"] = item["website"].replace("eurochange.co.uk", "nmmoney.co.uk")
+        yield from self.inspect_item(item, response)
```

Comment:
Looks like they are using `%H.%M` not `%H:%M`
```suggestion
        item["opening_hours"] = OpeningHours()
        item["opening_hours"].from_linked_data(ld_data, time_format="%H.%M")
        yield item
```